.github/workflows/pr-request-review.yml

```diff
@@ -0,0 +1,49 @@
+name: Request Review
+
+on:
+  issue_comment:
+    types:
+      - created
+
+jobs:
+  request-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if command is '/request_review'
+        id: check-command
+        run: |
+          if [[ $(echo "${{ github.event.comment.body }}" | tr '[:upper:]' '[:lower:]') == "/request_review"* ]]; then
```

Comment:
Lets use this check but with the `/request_review` command and set this for the whole job condition instead of a step because if the command does not match, the job should not run.

https://github.com/cryostatio/cryostat/blob/8c6c6a12b33e146fb9689251673e4d336a33bb06/.github/workflows/pr-ci.yml#L15